.action_templates/e2e-fork-template.yaml

```diff
@@ -0,0 +1,25 @@
+name: Run E2E Fork
+jobs:
+  - template: setup
+    if: github.event.pull_request.head.repo.full_name != 'mongodb/mongodb-kubernetes-operator' && contains(github.event.pull_request.labels.*.name, 'safe-to-test')
```

Comment:
the condition for forks, we run only if it has been labeled and is coming from a different repo.